### ISSUE TITLE
Error_XD

### ISSUE THREAD

**Issue by Pavlo-Minkievych** [avocode/avocode#2069]
---
* **Version**: 3.1.0
* **Sketch plugin version**: 50.0.0
* **Web/Desktop**: Desktop
* **Platform**: darwin
* **Avocode UID**: 68767
---
```
TypeError: Method Date.prototype.toISOString called on incompatible receiver #<Object>
    at Object.toISOString (<anonymous>)
    at Object.toJSON (<anonymous>)
    at JSON.stringify (<anonymous>)
    at stringify (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/raven-js/vendor/json-stringify-safe/stringify.js:23:15)
    at Raven._makeRequest (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/raven-js/src/raven.js:1924:39)
    at Raven._sendProcessedPayload (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/raven-js/src/raven.js:1877:52)
    at Raven._send (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/raven-js/src/raven.js:1811:14)
    at Raven._processException (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/raven-js/src/raven.js:1568:10)
    at Raven._handleStackInfo (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/raven-js/src/raven.js:1442:10)
    at Raven.captureExcep
```

**Comment by jakubzitny** [comment]
Hi @Pavlo-Minkievych, thanks for reporting this. We've seen this a few times in our logs already but we can't reproduce it at all. Can you please share more info on that? I believe it happened when syncing a design from XD, right? Was Avocode already open when you synced from XD? Can you reproduce this all the time or is it random? Or did it happen only this one time?

Thanks a lot.

**Comment by jakubzitny** [comment]
Hi @Pavlo-Minkievych, thanks for reporting this. We've seen this a few times in our logs already but we can't reproduce it at all. Can you please share more info on that? I believe it happened when syncing a design from XD, right? Was Avocode already open when you synced from XD? Can you reproduce this all the time or is it random? Or did it happen only this one time?

Thanks a lot.

**Comment by jakubzitny** [comment]
Hi @Pavlo-Minkievych, @smiley-cz and @Djamshid, this problem is now fixed in latest Avocode `3.3.1`, please update and let us know if you still have this or any other problems 😉
